examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -164,6 +165,24 @@ <h3>Image that is NOT blocked by consent</h3>
         </amp-consent>
         <!-- End didomi example -->
 
+      <!-- Marfeel example -->
+      <amp-consent id='marfeelConsent' layout='nodisplay' type='Marfeel'>
+        <script type="application/json">
+          {
+            "postPromptUI": "postPromptUI",
+            "clientConfig": {
+              "consentLanguage": "en",
+              "publisherLogo": "https://marfeel.mgr.consensu.org/amp/img/marfeel-logo.png",
```

Comment:
nit, please remove the trailing `,` so it's a valid json object. Thanks

Reply:
Fixed